src/components/SidePanel/SidePanelOverlay/index.tsx

```diff
@@ -1,14 +1,16 @@
 import { useSidePanelContext } from '../../../contexts/SidePanel'
 import { SidePanelOverlayStyled } from './style'
+import { SidePanelContentPropsDiv } from '../SidePanelContent/types'
 
-const SidePanelOverlay = () => {
+const SidePanelOverlay = (props: SidePanelContentPropsDiv) => {
```

Comment:
Are these new properties really being used? I didn't understand the reason.

Reply:
This is to have intellisense